nixos/tests/k3s/etcd.nix

```diff
@@ -0,0 +1,101 @@
+import ../make-test-python.nix ({ pkgs, lib, k3s, ... }:
+
+{
+  name = "${k3s.name}-etcd";
+
+  nodes = {
+
+    etcd = { ... }: {
+      services.etcd = {
+        enable = true;
+        openFirewall = true;
+        listenClientUrls = [ "http://192.168.1.1:2379" "http://127.0.0.1:2379" ];
```

Comment:
Listening to 0.0.0.0 won't be better?

Reply:
Better I don't know, but that would work too. I don't think there is a right answer here.